tools/node.js

```diff
@@ -23,6 +23,37 @@ new Function("exports", function() {
     return code.join("\n\n");
 }())(exports);
 
+function to_comment(value) {
+    if (typeof value != "string") value = JSON.stringify(value, function(key, value) {
+        return typeof value == "function" ? "<[ " + value + " ]>" : value;
+    }, 2);
+    return "// " + value.replace(/\n/g, "\n// ");
+}
+
+if (process.env["UGLIFY_BUG_REPORT"]) exports.minify = function(files, options) {
```

Comment:
Can you use a `+` on the env var value?
```js
if (+process.env["UGLIFY_BUG_REPORT"]) exports.minify = function(files, options) {
```
So that setting the env var to `"0"` will disable the bug report:

```
$ echo 'console.log(require(".").minify("console.log(1 + 2)", {}));' | node
{ code: 'console.log(3);' }
```
```
$ echo 'console.log(require(".").minify("console.log(1 + 2)", {}));' | UGLIFY_BUG_REPORT=0 node
{ code: 'console.log(3);' }
```
```
$ echo 'console.log(require(".").minify("console.log(1 + 2)", {}));' | UGLIFY_BUG_REPORT=1 node
{
  code: '// UGLIFY_BUG_REPORT\n' +
    '// {}\n' +
    '\n' +
    '//-------------------------------------------------------------\n' +
    '// INPUT CODE\n' +
    'console.log(1 + 2)'
}
```
It's sometimes easier to use `0` than unsetting the env var.

I won't mention handling "true", "false", "yes", "no", "on" and "off" as expected.  ;-)

Reply:
Added in https://github.com/mishoo/UglifyJS/pull/4516/commits/cfcf87f5e9b8023dfa624a9ac7f972a2e94574d9